Dockerfile.builder

```diff
@@ -21,25 +20,45 @@ RUN set -ex \
   done
 
 #ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.5.0
+ENV NODE_VERSION 6.9.2
```

Comment:
my bad, i did a rebase to master, and that pulled in these changes. @ldimaggi you should be ok to drop all this and only cherry pick the required 2 commits